docs/providers/aws/events/http-api.md

```diff
@@ -443,3 +443,13 @@ provider:
   httpApi:
     disableDefaultEndpoint: true
 ```
+
+### Service Naming
+
+You can use the `${service}-${stage}` naming for HTTP Gateway instead the default option `${stage}-${service}`.
```

Comment:
Let's rewrite this line as

```
You can use the `shouldStartNameWithService` option to change the naming scheme for HTTP API from the default `${stage}-${service}` to `${service}-${stage}`.
```

Reply:
No worries, this is clearer. It's done, thanks!